examples/ignition/k8s-master.yaml

```diff
@@ -196,7 +195,6 @@ storage:
       filesystem: rootfs
       contents:
         inline: |
-          FLANNELD_IFACE={{.ipv4_address}}
```

Comment:
Flannel uses the interface of the default route by default. https://github.com/coreos/flannel/issues/470